src/GitLabApiClient/GitLabClient.cs

```diff
@@ -92,10 +92,9 @@ public Task<Session> LoginAsync(string username, string password)
 
         private static string FixBaseUrl(string url)
         {
-            if (!url.EndsWith("/", StringComparison.OrdinalIgnoreCase))
-                url += "/";
+            url.TrimEnd('/');
```

Comment:
I think this does not work as intended, because TrimEnd does not mutate url local variable

Reply:
updated the tests to include the logic for FixBaseUrl